### ISSUE TITLE
p-Calendar don't update input on [dateFormat] changes

### ISSUE THREAD

**Issue by weltonborges** [primefaces/primeng#6197]
[x] bug report

p-Calendar don't update input on [dateFormat] changes

*the update occurs only after focus and blur

<p-calendar [locale]="locale" [dateFormat]="dateFormat" ... />

Angular version: 6.1.0
PrimeNG version: 6.0.2
Browser: all

**Comment by cagataycivici** [comment]
I'm unable to replicate with the latest PrimeNG. If the issue still persists with the new version please create a new issue with our stackblitz template.
